.github/workflows/test.yml

```diff
@@ -34,13 +35,25 @@ jobs:
           pip3 install lxml
 
       - name: Test nodl
-        uses: ros-tooling/action-ros-ci@0.1.0
+        uses: ros-tooling/action-ros-ci@v0.2
         id: action_ros_ci_step
         with:
           package-name: nodl_python ros2nodl
-          colcon-mixin-name: coverage-pytest
+          colcon-defaults: |
+            {
+              "build": {
+                "mixin": ["coverage-pytest"]
+              },
+              "test": {
+                "mixin": ["coverage-pytest"]
```

Comment:
Is it necessary to have it for both build & test?

Reply:
Yes it is.
[here](https://github.com/colcon/colcon-mixin-repository/blob/master/coverage.mixin#L14) you can see what it does (redirect the output + generating coverage files).

The output without specifying the test mixin: https://github.com/Guillaumebeuzeboc/nodl/actions/runs/3233376461/jobs/5295149298#step:5:19298
The output with the test coverage-pytest: https://github.com/ubuntu-robotics/nodl/actions/runs/3203856167/jobs/5234448783#step:5:19509